commands

```diff
@@ -31,8 +31,13 @@ case "$1" in
     touch "$LINKS_FILE"
 
     dokku_log_info1 "Starting container"
+    if [[ -n $POSTGRES_DOCKER_ARGS ]]; then
+      echo "$POSTGRES_DOCKER_ARGS" | tr ';' "\n" > "$SERVICE_ROOT/ENVIRONMENT"
+    else
+      echo "" > "$SERVICE_ROOT/ENVIRONMENT"
+    fi
     SERVICE_NAME=$(get_service_name "$SERVICE")
-    ID=$(docker run --name "$SERVICE_NAME" -v "$SERVICE_ROOT/data:/var/lib/postgresql/data" -e "POSTGRES_PASSWORD=$password" -d --restart always --label dokku=service --label dokku.service=postgres "$PLUGIN_IMAGE:$PLUGIN_IMAGE_VERSION")
+    ID=$(docker run --name "$SERVICE_NAME" -v "$SERVICE_ROOT/data:/var/lib/postgresql/data" --env-file="$SERVICE_ROOT/ENVIRONMENT" -e "POSTGRES_PASSWORD=$password" -d --restart always --label dokku=service --label dokku.service=postgres "$PLUGIN_IMAGE:$PLUGIN_IMAGE_VERSION")
```

Comment:
I think all is well, only thing is the name of the variable. Since it’s really only env vars, it’ll make more sense to name it something like `POSTGRES_CUSTOM_ENV` or something better :)
@josegonzalez @mcolyer thoughts?